sphinx_intl/basic.py

```diff
@@ -68,6 +68,7 @@ def update(locale_dir, pot_dir, languages, line_width=76):
                 else:  # new po file
                     status['create'] += 1
                     click.echo('Create: {0}'.format(po_file))
+                    cat_pot.locale = lang
```

Comment:
Note: Surely, assigning a string to `Catalog.locale` will generate the `Language` header (through `locale_identifier` attribute)
https://github.com/python-babel/babel/blob/7ebdf5a249ff8861bbcd3f7e4cc1ba9555c3fb65/babel/messages/catalog.py#L398-L399